docs/resources/security_identifier.md.erb

```diff
@@ -0,0 +1,63 @@
+---
+title: About the security_identifier Resource
+platform: windows
+---
+
+# security_identifier
+
+Use the `security_identifier` InSpec resource to test the Security Identifier (SID) for users and groups on Windows..
+
+<br>
+
+## Availability
+
+### Installation
+
+This resource is distributed along with InSpec itself. You can use it automatically.
+
+### Version
+
+This resource first became available in vX.Y.Z of InSpec.
+
+## Syntax
```

Comment:
Prefer 'Resource Parameters' as a header instead of 'Syntax'.  We're inconsistent about this :/